src/main/java/com/netflix/simianarmy/client/aws/chaos/VlabChaosCrawler.java

```diff
@@ -0,0 +1,101 @@
+package com.netflix.simianarmy.client.aws.chaos;
+
+
+import com.amazonaws.services.ec2.model.Instance;
+import com.netflix.simianarmy.GroupType;
+import com.netflix.simianarmy.chaos.ChaosCrawler;
+import com.netflix.simianarmy.client.aws.AWSClient;
+import com.netflix.simianarmy.vlabs.chaos.VlabsInstanceGroup;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.EnumSet;
+import java.util.LinkedList;
+import java.util.List;
+
+
+/**
+ * The Class VlabChaosCrawler. This will crawl for all available EC2 Instance associated with a DeepField VLab name.
+ */
+public class VlabChaosCrawler implements ChaosCrawler {
```

Comment:
I"m wondering if we should use more generic names here, cause it's not specific to vlabs, it's just a name pattern lookup, right? Since the is a fork, and is public, we'll want to leave out as much environment specific detail as possible.

Reply:
I agree this is a bad name. I'll update to be more generic like `InstanceGroupCrawler`